### PR TITLE
"istioctl pc route" output add "VHOST NAME"

### PR DESCRIPTION
**Pull request by zengyuxing007**
**Please provide a description of this PR:**

ref: #44413

### CODE REVIEW

File: istioctl/pkg/writer/envoy/configdump/route.go [hunk 1]
```diff
@@ -63,16 +63,18 @@ func (c *ConfigWriter) PrintRouteSummary(filter RouteFilter) error {
 				for _, vhosts := range route.GetVirtualHosts() {
 					for _, r := range vhosts.Routes {
 						if !isPassthrough(r.GetAction()) {
-							fmt.Fprintf(w, "%v\t%s\t%s\t%s\n",
+							fmt.Fprintf(w, "%v\t%s\t%s\t%s\t%s\n",
```
Comment (hanxiaop) [inline istioctl/pkg/writer/envoy/configdump/route.go:66]: Do you have any sample output of this?

Comment (zengyuxing007) [reply istioctl/pkg/writer/envoy/configdump/route.go:66]:
```shell
/tmp/istioctl-linux-amd64  pc route istio-ingressgateway-68d5f6f757-w7g4z -n istio-system
NAME                                        VHOST NAME                    DOMAINS                   MATCH                  VIRTUAL SERVICE
https.443.https.httpbin-gateway.default     httpbin.quic-corp.com:443     httpbin.quic-corp.com     /*                     httpbin-route.default
http.80                                     *:80                          *                         /productpage           bookinfo.default
http.80                                     *:80                          *                         /static*               bookinfo.default
http.80                                     *:80                          *                         /login                 bookinfo.default
http.80                                     *:80                          *                         /logout                bookinfo.default
http.80                                     *:80                          *                         /api/v1/products*      bookinfo.default
http.80                                     *:80                          *                         /echo*                 bookinfo.default
http.80                                     *:80                          *                         /test                  ingressgateway.default
http.80                                     *:80                          *                         /mock                  ingressgateway.istio-system
http.80                                     *:80                          *                         /mock                  ingressgateway.istio-system
http.80                                     *:80                          *                         /mock                  ingressgateway.istio-system
                                            backend                       *                         /stats/prometheus*
                                            backend                       *                         /healthz/ready*
```

Comment (zengyuxing007) [reply istioctl/pkg/writer/envoy/configdump/route.go:66]: The other example is for a simple pod that has been injected with sidecar; if the single line of output exceeds the width of the terminal, you may need to adjust the terminal's font size or display resolution for a better experience.

image

**Review by hanxiaop — APPROVED** [review]
LGTM, thanks

**Pull request merged**